src/components/Toolbar/MenuBar/MenuBar.js

```diff
@@ -150,7 +150,7 @@ export const MenuBar = ({
     }
 
     const onSaveComplete = (res, copy = false) => {
-        if (res.status === 'OK' && res.response.uid) {
+        if (res.response.uid) {
```

Comment:
no idea what `res.status` used to do, but why don't we need `res.status === 'OK'` anymore?

Reply:
@HendrikThePendric  pointed out to me that this is a callback passed to `onComplete` in the mutation, which is called only on successful mutation.
Also, `res` sometimes can be not defined even for successful mutations.

Reply:
This is being used inside the `onComplete` callback of a `useDataMutation` hook, so the res must be OK. Otherwise we'd be in the `onError`.